backend/btrixcloud/pages.py

```diff
@@ -304,7 +304,7 @@ async def add_page_note(
         if not result:
             raise HTTPException(status_code=404, detail="page_not_found")
 
-        return {"added": True}
+        return {"added": True, "data": note}
```

Comment:
@tw4l I noticed sometimes we return the created resource ID, might be nice to standardize around returning the entire resource as `data`? Definitely open to suggestions here.

Reply:
I like this! I think we've mostly returned the created resource ID where we expect to need it somewhat imminently for the frontend or tests. But your suggestion here is much cleaner. I'll make an issue about cleaning this up/standardizing more broadly.

Reply:
Issue: https://github.com/webrecorder/browsertrix/issues/1724